tests/apollo/test_skvbc_reconfiguration.py

```diff
@@ -1797,7 +1797,7 @@ async def test_install_command(self, bft_network):
     
 
     async def try_to_unwedge(self, bft_network, bft, restart, quorum=None):
-        with trio.fail_after(seconds=60):
+        with trio.fail_after(seconds=90):
```

Comment:
nit: why did you have to increase it?

Reply:
This will be reverted as all the reconfiguration testcases are fixed in another PR. Once that goes in I will merge and commit.